src/server/modules/user/interfaces/UrlManagementService.ts

```diff
@@ -25,6 +27,9 @@ export interface UrlManagementService {
   getUrlsWithConditions: (
     conditions: UserUrlsQueryConditions,
   ) => Promise<UrlsPaginated>
+  getTagsWithConditions: (
```

Comment:
ditto as above comment, should we move this out into a separate `TagManagementService`

Reply:
let's discuss offline